samples/core/QueryTypes/QueryTypes.csproj

```diff
@@ -1,11 +1,11 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
     <RootNamespace>Samples</RootNamespace>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="2.1.0-*" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="2.1.0-*" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="3.0.0-preview9.19423.6" />
```

Comment:
It is preferable to use wildcards than to tie this to preview 9.

Reply:
Maybe we can just do a pass over all samples for version updates later (https://github.com/aspnet/EntityFramework.Docs/issues/1074).